flink-connectors/flink-jdbc/src/test/java/org/apache/flink/api/java/io/jdbc/catalog/PostgresCatalogTestBase.java

```diff
@@ -58,6 +58,7 @@
 	protected static final String TABLE5 = "t5";
 	protected static final String TABLE_PRIMITIVE_TYPE = "dt";
 	protected static final String TABLE_ARRAY_TYPE = "dt2";
+	protected static final String TABLE_SERIAL_TYPE = "dt3";
```

Comment:
why use a separate table rather than reusing existing one?

Reply:
Beacause the other table was becoming too big..I think we need to split it a little bit..what do you think?

Reply:
I'd recommend to rename dt,dt2, and dt3 to reflect the types they individually contain and test; otherwise, it's getting confusing

Reply:
How should I rename them? Do you have something in mind?

Reply:
”primitive_type“, "array_type", "serial_type"?